src/diagnostics/structs/CommandRunResult.ts

```diff
@@ -10,12 +12,22 @@ export class CommandRunResult<ResponseType extends { error?: {}, data?: {} }> {
     public get json(): ResponseType {
         if (this._json === undefined) {
             try {
-                this._json = JSON.parse(this.output);
+                this._json = JSON.parse(this._stdout);
             } catch (err) {
                 this._json = null;
             }
         }
 
         return this._json;
     }
+
+    private parseStdOut(): void {
+        try {
+            const outputJson = JSON.parse(this.output);
+
+            this._stdout = outputJson.stdout;
+        } catch (err) {
+            this._stdout = null;
```

Comment:
is it useful to log something when we can't parse stdout?

Reply:
Agree. This is what https://github.com/Microsoft/vscode-azureappservice/issues/194 is talking about. This class is used in both the extension host and the debug adapter. Their respective logging method is very different. So I will have to make it a separate task, refactor and unify them then.